client/reader/recent/style.scss

```diff
@@ -76,11 +76,5 @@
 
 	&__post-column {
 		flex: 3;
-
-		.reader-full-post__sidebar,
-		.reader-full-post__author-block,
-		.back-button {
-			display: none;
-		}
```

Comment:
This is no longer needed because we now exclude these elements if the `layout === recent`.